pom.xml

```diff
@@ -204,6 +204,12 @@
       <version>9.1.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc8</artifactId>
```

Comment:
@harawata Don't you want the java 11 version here?

Reply:
Good catch 👍
I'll update later.